src/server/services/afis/afis-facturen.ts

```diff
@@ -273,28 +274,24 @@ function transformFacturen(
   };
 }
 
-function shouldFilterOutFactuur(postingDate: string): boolean {
-  const postingDateAsDate = new Date(postingDate);
+function isDownloadAvailable(postingDate: string): boolean {
+  if (!postingDate) {
+    return true;
+  }
+
+  const datePosted = parseISO(postingDate);
   const now = new Date();
 
-  if (postingDateAsDate.getTime() > now.getTime()) {
-    captureMessage('postingDate of invoice is in the future', {
-      severity: 'error',
-    });
+  if (datePosted.getTime() > now.getTime()) {
     return false;
   }
 
-  const postedToday =
-    postingDateAsDate.getDate() === now.getDate() &&
-    postingDateAsDate.getMonth() === now.getMonth() &&
-    postingDateAsDate.getFullYear() === now.getFullYear();
-
-  if (!postedToday) {
-    return true;
+  if (isToday(datePosted)) {
+    const sevenOClock = 19;
```

Comment:
Dit zegt niet zoveel over de waarde 19 toch? Als het 8 uur wordt.. `sevenOClock = 20` ?

Reply:
Deze is aangepast na derhand van een vorige comment